lib/capistrano/tasks/drupal/jsonapi.rake

```diff
@@ -0,0 +1,30 @@
+##
+# JSON:API Recipes
+#
+# Manage your Drupal Json API.
+# @see https://www.drupal.org/docs/8/core/modules/jsonapi-module
+##
+
+namespace 'drupal:jsonapi' do
+  namespace :explorer do
+
+    desc "Enable JSON:API Explorer"
+    task :enable do
+      on roles(:app) do
+        within release_path.join(fetch(:app_path)) do
+          execute :drush, "en", "jsonapi_explorer", "-y"
```

Comment:
We should offer a more generic ways to enable/ disable modules

Reply:
I agree, it could be on capdrupal